1-Authentication/1-sign-in/README.md

```diff
@@ -0,0 +1,365 @@
+
+MSAL React is a wrapper around the [Microsoft Authentication Library for JavaScript](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-browser) (MSAL.js). As such, it exposes the same public APIs that MSAL.js offers, while adding many new features customized for modern React applications.
+
+Here you'll learn how to [sign-in](https://docs.microsoft.com/azure/active-directory/develop/scenario-spa-sign-in), [acquire a token](https://docs.microsoft.com/azure/active-directory/develop/scenario-spa-acquire-token) and [call a protected web API](https://docs.microsoft.com/azure/active-directory/develop/scenario-spa-call-api), as well as how to work with different [audiences and account types](https://docs.microsoft.com/azure/active-directory/develop/v2-supported-account-types).
+
+## Scenario
+
+1. The client React SPA uses the Microsoft Authentication Library (MSAL) to sign-in a user and obtain a JWT [access token](https://docs.microsoft.com/azure/active-directory/develop/access-tokens) from **Azure AD**.
+2. The access token is used as a *bearer token* to authorize the user to call the **Microsoft Graph API**.
+
+![Overview](./ReadmeFiles/topology.png)
+
+## Contents
+
+| File/folder         | Description                                      |
+|---------------------|--------------------------------------------------|
+| `App/App.jsx`       | Main application logic resides here.             |
+| `App/authConfig.js` | Contains authentication parameters.              |
+| `App/ui.jsx`        | UI update logic resides here.                    |
+| `App/fetch.jsx`     | Provides a helper method for making fetch calls. |
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/en/download/) must be installed to run this sample.
+- [Visual Studio Code](https://code.visualstudio.com/download) is recommended for running and editing this sample.
+- A modern web browser. This sample uses **ES6** conventions and will not run on **Internet Explorer**.
+- An **Azure AD** tenant. For more information see: [How to get an Azure AD tenant](https://docs.microsoft.com/azure/active-directory/develop/quickstart-create-new-tenant)
+- A user account in your **Azure AD** tenant. This sample will not work with a **personal Microsoft account**. Therefore, if you signed in to the [Azure portal](https://portal.azure.com) with a personal account and have never created a user account in your directory before, you need to do that now.
+
+## Setup
+
+### Step 1: Clone or download this repository
+
+From your shell or command line:
+
+```console
+    git clone https://github.com/Azure-Samples/ms-identity-javascript-react-tutorial.git
+```
+
+or download and extract the repository .zip file.
+
+> :warning: To avoid path length limitations on Windows, we recommend cloning into a directory near the root of your drive.
+
+### Step 2: Install project dependencies
+
+```console
+    cd ms-identity-javascript-react-tutorial
+    cd ms-identity-react-c1s1
```

Comment:
Different folder structure: cd 1-Authentication\1-sign-in\App

Reply:
+1

---
In reply to: [574133766](https://github.com/Azure-Samples/ms-identity-javascript-react-tutorial/pull/1#discussion_r574133766) [](ancestors = 574133766)